src/csvToJson.js

```diff
@@ -1,36 +1,19 @@
-/* eslint-disable no-unused-vars */
 import { write, readDir } from '@groceristar/static-data-generator';
-// import { joinPath } from './utils';
-import fs from 'fs';
-import parseCsv from './parseCsv';
 
-function makeFullPath(pathData, filetype) {
```

Comment:
when you removing makeFullPath, did you check if it not called at any places, right?

Reply:
yes